packages/frontend/src/hooks/useRefreshServer.tsx

```diff
@@ -103,7 +103,13 @@ export const useJob = (
         queryKey: ['job', jobId],
         queryFn: () => getJob(jobId || ''),
         enabled: !!jobId,
-        refetchInterval: (data) => data?.jobStatus === 'RUNNING' && 500,
+        refetchInterval: (data) =>
+            [
+                JobStatusType.DONE.valueOf(),
+                JobStatusType.ERROR.valueOf(),
```

Comment:
I don't think we need `.valueOf()`

Reply:
thanks! the logic was also a bit wrong. Before it would set to false if `data` was undefined but in my case it was going to `500`